jb/itch5/order_book_depth.hpp

```diff
@@ -122,29 +130,45 @@ class order_book {
    * @param side the side of the book to update
    * @param px the price of the order that was reduced
    * @param reduced_qty the quantity reduced in the order
-   * @returns true if the inside changed
-   */
+   * @return true if the inside changed
+   * Decrements order_book_ if a price (px) is erased from the map
+   *
+   * Check first if the px level exists. The use of find is ok since 
+   * the price level "has" to exist, and the iterator returned is used 
+   * to update the quantity (or erase it if qty is 0 or less)
+   * The following exceptions (no Throw executed) are logged as warnings
+   * into JOB_LOG
+   * EXC1: trying to reduce a non-existing price level
```

Comment:
I think you can document exceptions with \throws.

https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdthrows